angular-workspace/projects/ni/nimble-angular/pipes/duration.pipe.ts

```diff
@@ -5,7 +5,8 @@ import { DurationFormatter } from '@ni/nimble-components/dist/esm/table-column/d
  * A pipe that transforms a number of milliseconds (string/number) into a string formatted like "1 day, 2 hr, 3 min, 4.567 sec"
  */
 @Pipe({
-    name: 'duration'
+    name: 'duration',
+    standalone: true
```

Comment:
This may be controversial since it is an unnecessary breaking change, but it seems preferable to wrapping all our pipes together in one module.